config/webpack/development.js

```diff
@@ -1,6 +1,6 @@
 const environment = require('./environment')
 
 const config = environment.toWebpackConfig()
-config.devtool = 'sourcemap';
+// config.devtool = 'sourcemap';
```

Comment:
I think we commented this out to debug something, can we leave it in here for this PR?